machineconfiguration/v1alpha1/types_pinnedimageset.go

```diff
@@ -26,6 +26,20 @@ type PinnedImageSet struct {
 	// spec describes the configuration of this pinned image set.
 	// +kubebuilder:validation:Required
 	Spec PinnedImageSetSpec `json:"spec"`
+
+	// status describes the last observed state of this pinned image set.
+	// +optional
+	Status PinnedImageSetStatus `json:"status"`
```

Comment:
missing status subresource in `// +` tags above?

Reply:
gotcha kubebuilder adding

```
// +kubebuilder:subresource:status
```